vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -41,6 +42,20 @@ def ems_inv_to_hashes
 
       private
 
+      def all_server_resources
+        return @all_server_resources if @all_server_resources
+
+        resources = []
+        stacks.each do |stack|
+          # Nested depth 50 just for sure, although nobody should nest templates that much
```

Comment:
I will still keep this. After we will merge the nested stacks support, this should be refactored to leverage that. Right now it is just easier to call give me all nested resources.